modules/admin-ui/src/main/java/org/opencastproject/adminui/endpoint/AbstractEventEndpoint.java

```diff
@@ -1164,29 +1164,32 @@ public Response getBulkConflicts(@FormParam("update") final String updateJson) t
     final BulkUpdateUtil.BulkUpdateInstructions instructions;
     try {
       instructions = new BulkUpdateUtil.BulkUpdateInstructions(updateJson);
-    } catch (IllegalArgumentException e) {
+    } catch (final IllegalArgumentException e) {
```

Comment:
@KatrinIhler We expect lectures to take place on regular intervals. For example, on each Monday at 10am to 12am during the semester. You also see this in the Add Event wizard that allows you to create events on rrules (in particular, by setting a week day).

It is true that this might seem arbitrary if you select an arbitrary set of events. But in real world use cases, selecting arbitrary events to change their scheduling won't be of any use anyway: Unrelated events will not have anything in common you would want to change at once (neither room, week day, start time or any other value you can edit with this function).

Example:
The lecture Mathematics was scheduled to take place starting in the first week of the semester until end of semester (specified in date) on Mondays, 10am-11am, Room A and Thursdays, 1pm-3pm, Room B.

The reason why people would want to edit scheduled events (as far as I understood) are changes like: 

Later, the schedule has changed: The lecture Mathematics takes place in Room X on Mondays, and Room B at 2pm-4pm on Fridays.

That kind of changes can now easily be done because of the grouping by week days.

If not grouped by weekdays, the user would need to select all events of the lecture that take place on Thursdays (that is error prone) to be able to change the week day.

This way, the user can filter for a series and just select all events of the series. This works quite well and it the expected use case for this feature.

Reply:
@pmiddend I don't think we have a guide for that. In general I try to be consistent with the existing code unless it makes no sense at all (though that is sometimes not consistent with itself, then I choose the option I like better...) I would use final to inform the reader that a variable cannot and will not be reassigned, but since e is only visible in this block and no sane person would reassign it I would leave it out here.

Reply:
Okay, fixed that.